home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/InvitationDto.java

```diff
@@ -0,0 +1,22 @@
+package com.softserveinc.ita.homeproject.homeservice.dto;
+
+import java.time.LocalDateTime;
+import com.softserveinc.ita.homeproject.homedata.entity.Role;
+
+import lombok.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
```

Comment:
Why do we need it?

Reply:
When I wrote test and tried to compare DTOs I discovered that equals doesn't work. That's why I've added it.

Reply:
Ok